srcpkgs/arb/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'arb'
+pkgname=arb
+version=2.19.0
+revision=1
+build_style="configure"
+configure_args="--prefix=/usr --with-gmp=/usr --with-mpfr=/usr --with-flint=/usr"
+makedepends="flintlib-devel"
+short_desc="C library for arbitrary-precision interval arithmetic"
+maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
+license="LGPL-2.1-or-later"
+homepage="https://arblib.org"
+distfiles="https://github.com/fredrik-johansson/arb/archive/${version}.tar.gz"
+checksum=0aec6b492b6e9a543bdb3287a91f976951e2ba74fd4de942e692e21f7edbcf13
+nocross=yes
```

Comment:
Do you have  a simple comment you can add here explaining why?

Reply:
I don't remember from the top of my head; I did try it and something didn't work. I will retry it and comment.

Reply:
- ntl has a custom configure perl script which runs binaries built for target.
- flintlib and arb should cross-compile fine once ntl is packaged